### ISSUE TITLE
Gonevim crashes when typing command

### ISSUE THREAD

**Issue by Yohannfra** [akiyosi/goneovim#101]
Gonevim --version : (also crashes) 
```
open /home/yohann/.goneovim/setting.toml: no such file or directory
msgpack/rpc: session closed
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x971276]

goroutine 51 [running]:
github.com/neovim/go-client/nvim.(*Nvim).call(0x0, 0xd07993, 0xc, 0x0, 0x0, 0xc0002d3f10, 0x1, 0x1, 0xc0003a6000, 0xe1b420)
        github.com/neovim/go-client/nvim/nvim.go:406 +0x26
github.com/neovim/go-client/nvim.(*Nvim).Command(0x0, 0xcffe36, 0x3, 0xc000240240, 0xbb823350)
        github.com/neovim/go-client/nvim/apiimp.go:740 +0xb6
created by github.com/akiyosi/goneovim/editor.(*MiniMap).exit
        github.com/akiyosi/goneovim/editor/minimap.go:157 +0x5b
```

It crashes when i type ```:GonvimGridFont 4``` (4 or 2, 3, etc..)
here is the backtrace
```
open /home/yohann/.goneovim/setting.toml: no such file or directory
panic: interface conversion: interface {} is int64, not string

goroutine 1 [running, locked to thread]:
github.com/akiyosi/goneovim/editor.(*Screen).gridFont(0xc0002e96c0, 0xc0d720, 0xc0004637a0)
        github.com/akiyosi/goneovim/editor/screen.go:387 +0x87b
github.com/akiyosi/goneovim/editor.(*Workspace).handleRPCGui(0xc000266000, 0xc0003b1060, 0x2, 0x2)
        github.com/akiyosi/goneovim/editor/workspace.go:1142 +0xc7d
github.com/akiyosi/goneovim/editor.(*Workspace).registerSignal.func2()
        github.com/akiyosi/goneovim/editor/workspace.go:238 +0x7d
github.com/akiyosi/goneovim/editor.callbackworkspaceSignal28652a_GuiSignal(0x2e10990)
        github.com/akiyosi/goneovim/editor/moc.go:1167 +0x68
github.com/akiyosi/goneovim/editor._cgoexpwrap_a074abc4c182_callbackworkspaceSignal28652a_GuiSignal(0x2e10990)
        _cgo_gotypes.go:2221 +0x2b
github.com/akiyosi/goneovim/editor._Cfunc_workspaceSignal28652a_EventDefault(0x2e10990, 0x7f03c80012c0, 0x0)
        _cgo_gotypes.go:1383 +0x4a
github.com/akiyosi/goneovim/editor.(*workspaceSignal).EventDefault.func1(0xc000256000, 0xe1b560, 0xc0002aaba0, 0xc0002aaba0)
        github.com/akiyosi/goneovim/editor/moc.go:1721 +0xb6
github.com/akiyosi/goneovim/editor.(*workspaceSignal).EventDefault(0xc000256000, 0xe1b560, 0xc0002aaba0, 0x5)
        github.com/akiyosi/goneovim/editor/moc.go:1721 +0x4d
github.com/akiyosi/goneovim/editor.callbackworkspaceSignal28652a_Event(0x2e10990, 0x7f03c80012c0, 0xd37a98)
        github.com/akiyosi/goneovim/editor/moc.go:1716 +0x11e
github.com/akiyosi/goneovim/editor._cgoexpwrap_a074abc4c182_callbackworkspaceSignal28652a_Event(0x2e10990, 0x7f03c80012c0, 0x7ffcb96b7dd0)
        _cgo_gotypes.go:2390 +0x35
github.com/therecipe/qt/widgets._Cfunc_QApplication_QApplication_Exec(0xc000000000)
        _cgo_gotypes.go:2479 +0x49
github.com/therecipe/qt/widgets.QApplication_Exec(...)
        github.com/therecipe/qt/widgets/widgets-minimal.go:2849
github.com/akiyosi/goneovim/editor.InitEditor()
        github.com/akiyosi/goneovim/editor/editor.go:238 +0x780
main.main()
        github.com/akiyosi/goneovim/cmd/goneovim/main.go:14 +0x20

```

**Comment by Yohannfra** [comment]
I'm using the version ```v0.4.4```

**Comment by akiyosi** [comment]
Hi, thank you for your reporting.
I fixed this issue in 9d52645b310e
